korio/src/commonMain/kotlin/com/soywiz/korio/file/std/StandardPathsBase.kt

```diff
@@ -42,7 +41,7 @@ interface StandardPathsBase {
      * - windows: /Users/user
      * - macos: /Users/user
      */
-    val userHome: String get() = Environment["~"] ?: "/tmp"
```

Comment:
Shouldn't be better that the getter performs the expansion?

Reply:
I don't know what you mean :) could you please elaborate?